dbfit-java/core/src/main/java/dbfit/util/DbParameterAccessor.java

```diff
@@ -78,15 +83,29 @@ public void bindTo(StatementExecution cs, int ind) throws SQLException{
         this.cs=cs;
         this.index=ind;    
         if (direction != INPUT){
-            cs.registerOutParameter(ind, getSqlType());
+            cs.registerOutParameter(ind, getSqlType(), direction == RETURN_VALUE);
         }
     }
 
+    private Object transformValue(Object value) throws SQLException {
```

Comment:
Yes, makes sense. I like `transformToJdbcType` or `transformDbfitToJdbcType`, aligned with the 'Type' part of the class and interface names. 

But do we use "JDBC" (as we have in some field/variable names) or "Jdbc"?